ci/prow/config.yaml

```diff
@@ -113,7 +113,9 @@ presubmits:
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/test-account/service-account.json
         - name: E2E_CLUSTER_REGION
-          value: us-central1
+          value: us-west1
+        - name: E2E_CLUSTER_BACKUP_REGIONS
+          value: us-east1 us-central1
```

Comment:
do they need to be quoted single there is a space?

Reply:
I think so.

Reply:
Done.